src/adhocracy_pcompass/adhocracy_pcompass/resources/subscriber.py

```diff
@@ -0,0 +1,72 @@
+"""Sync external ressources with elastic search."""
+
+import re
+import requests
+import logging
+
+from substanced.util import find_service
+from pyramid.traversal import find_interface
+
+from adhocracy_core.interfaces import IResourceCreatedAndAdded, search_query
```

Comment:
Ok PEP8 is on your side, I read it too quickly/wrongly but `python_guidelines.rst` says "one import per line". Maybe because it's easier to grep? Anyway that's what we have everywhere.